### PR TITLE
Search UX Refinements

### PR DESCRIPTION
**Pull request by BenJam**
* remove explore button (will add it back to the homepage in a later)
* remove search bar for search result page (which is for search results
only)
* remove some of the extraneous information from project list
* simplify search page layout to match `/unseen-infrastructure` et al)
* move the nav search bar left to make it more prominent
* add search query (if there is one) to the search bar in nav
* Make result ordering more obvious
* Make fitter results more obvious

screen shot 2017-05-18 at 15 19 36

### CODE REVIEW

**Comment by andrew** [comment]
@BenJam can you add a screenshot preview of the changes?

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -129,6 +129,10 @@ h3,h4{
   }
 }
 
+h4 a{
```
Comment (andrew) [inline app/assets/stylesheets/application.scss]: This is a bit too broad, suspect it will affect other areas of the site where we use links in h4 headings

Comment (BenJam) [inline app/assets/stylesheets/application.scss]: NP I was jus thinking I may throw them into panels. Will amend.

Comment (BenJam) [inline app/assets/stylesheets/application.scss]: h5 (h4 has a lot of spacing) elements for titles, any better? I'll leave the 'clear' until #542 
screen shot 2017-05-18 at 15 19 36

**Pull request merged**